src/processor/operator/aggregate/hash_aggregate.cpp

```diff
@@ -10,6 +10,26 @@ using namespace kuzu::storage;
 namespace kuzu {
 namespace processor {
 
+std::string HashAggregatePrintInfo::toString() const {
+    std::string result = "";
+    result += "Group By: [";
+    for (auto& expr : allKeys) {
```

Comment:
this logic can be refactor or reuse the function in ExpressionUtils::toString.

Because all of them are printing a list of expressions split by ','